app/javascript/hyacinth_ui_v1/components/digital_objects/children/Children.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
-import produce from 'immer';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
 
 import digitalObjectInterface from '../digitalObjectInterface';
+import AssetNew from '../new/AssetNew';
 import TabHeading from '../../ui/tabs/TabHeading';
+import GraphQLErrors from '../../ui/GraphQLErrors';
 
-class Children extends React.Component {
-  // state = {
-  //   digitalObject: {},
-  // }
-
-  // componentDidMount() {
-  //   const { match: { params: { id } } } = this.props;
-  //
-  //   digitalObject.get(id)
-  //     .then((res) => {
-  //       console.log(res.data);
-  //       this.setState(produce((draft) => {
-  //         draft.digitalObject = res.data.digitalObject;
-  //       }));
-  //     });
-  // }
-
-  render() {
-    const { data: { uid } } = this.props;
-
-    return (
-      <>
-        <TabHeading>Manage Child Assets</TabHeading>
-        <p>Child Digital Objects</p>
-        <p># Total</p>
-
-
-      </>
-    );
+const getChildAssetsQuery = gql`
+  query ChildAssets($id: ID!){
+    childAssets(id: $id) {
+      id
+    }
   }
-}
+`;
+
+const Children = (props) => {
+  const { match: { params: { id } } } = props;
+  const { loading, error, data, refetch } = useQuery(getChildAssetsQuery, { variables: { id } });
+
+  if (loading) return (<></>);
+  if (error) return (<GraphQLErrors errors={error} />);
+  return (
+    <>
+      <TabHeading>Manage Child Assets</TabHeading>
+      <Card className="mb-3">
+        <Card.Body>
+          <Card.Title>
+            Add New Asset
+          </Card.Title>
+          <AssetNew parentId={id} refetch={refetch} />
+        </Card.Body>
+      </Card>
+      <Card className="mb-3">
+        <Card.Body>
+          <Card.Title>
+            Child Digital Objects
+          </Card.Title>
+          {
+            data && (
+              data.childAssets.map(asset => (
+                <p><Link to={`/digital_objects/${asset.id}`}>{asset.id}</Link></p>
+              ))
+            )
+          }
+          <p>{`${data.childAssets.length} Total`}</p>
+        </Card.Body>
+      </Card>
+    </>
+  );
+};
+
+Children.propTypes = {
+  match: PropTypes.object.isRequired,
+};
 
 export default digitalObjectInterface(Children);
```

Comment:
This component will need some restructuring after your rebase, since lower case d digitalObjectInterface no longer exists.  If you search through the project code for `<DigitalObjectInterface` you'll see examples of how to use the new component.

Reply:
done per SystemData, etc.